src/main/java/apka/config.json

```diff
@@ -0,0 +1 @@
+{"gpsPort":1911,"gpsIP":"0.0.0.0","gprsIP":"0.0.0.1","port":7779,"ip":"0.0.0.1","gprsPort":9111}
```

Comment:
te moduły będą stały na losowych portach, o ich parametrach (ip, port, nazwa, metody) dowiesz się gdy wykonają one register. Jeżeli jest to do debugu to ok